tests/Etc/EarlyIdentification/Service.php

```diff
@@ -10,6 +10,6 @@ class Service
 
     public function __construct()
     {
-        $this->token = config('tenancy.token');
+        $this->token = config('tenancy.token', '');
```

Comment:
What's this change for? Specifically within this PR.

Reply:
`token` had to have a string value, so the service relied on `config('tenancy.token')` to be set. 

I think this isn't needed now – the model discovery discovered all instantiable classes in the tests/Etc directory, and that logic is a bit different now.